openslides/motions/views.py

```diff
@@ -66,6 +66,16 @@ def check_view_permissions(self):
             result = False
         return result
 
+    def retrieve(self, request, *args, **kwargs):
+        """
+        Customized view endpoint to retrieve a motion.
+
+        Adds the allowed actions for the motion.
+        """
+        response = super().retrieve(request, *args, **kwargs)
+        response.data['allowed_actions'] = self.get_object().get_allowed_actions(request.user)
+        return response
```

Comment:
I don't like this method. When I am looking for all fields for the motion, then I look in serlializers.py. Therefore this functionality is hard to debug.

If we need this method in python, then it should be a read only field on the motion serializer. But I would prefer that we move this function to js.

@normanjaeckel what do you think?